common/locales/en/settings.json

```diff
@@ -41,7 +41,7 @@
     "json": "(JSON)",
     "customDayStart": "Custom Day Start",
     "24HrClock": "24Hr Clock",
-    "clockInfo": "HabitRPG defaults to check and reset your Dailies at midnight each day. You can customize that here (Enter number between 0 and 24).",
+    "customDayStartInfo": "HabitRPG defaults to checking and resetting your Dailies at midnight each day. You can customize that here. Enter a number between 0 and 24. It is recommended that you first read <a href=\"http://habitrpg.wikia.com/wiki/Custom_Day_Start\" target=\"_blank\">this advice about how to change it</a>.",
```

Comment:
There's a few warnings that would need to be included (see [Custom Day Start](http://habitrpg.wikia.com/wiki/Custom_Day_Start) for them all). I'm trying to think how it would be best to include them. I think there's too many to display directly on the Settings screen. We could put the CDS field and all the warnings in a modal that's accessed from the Settings screen. Or leave the field as it is, and have "click here to show advice" text that causes the warnings to appear on the Settings page. (Either of those would be instead of linking to the wiki page.) Do you have a preference for how it should be done?